### DISCUSSION TITLE
emulation of bank switching cartridge ROMs

### DISCUSSION THREAD

**Discussion by banman11** [frntc/Sidekick64#27] [Q&A]
I have noticed that emulation of bank switching cartridge ROMs is supported in the documentation.
I am interested in getting the Super Zaxxon cartridge to run on the Sidekick64. I understand that there are 2 ROMs involved in the original hardware.
Does the Sidekick64 use 2 ROMs in the same fashion? 
That is to say are the !ROM hi and !ROM lo cartridge lines utilised in the same way as the original super Zaxxon cardridge does?

**Comment by frntc** [comment]
The emulation does implement the ROM LO and HI accesses and bank switching. Of course the bank switching logic/behavior (a 74LS74 and 74LS00 iirc) is reproduced in software. The emulation is probably not suited to produce the PLA glitches the Zaxxon cartridge is famous for.

**Comment by banman11** [comment]
Very interesting. I might further investigate.
By the way your Sidekick64 cartridge is fantastic! Thank you.
